source/authentication/email-password.txt

```diff
@@ -244,6 +257,11 @@ A user confirmation function is generally structured as follows:
       }
    }
 
+Unless the custom function automatically confirms the user, you must provide a 
```

Comment:
I think "you must provide a means for users to respond to the confirmation message that the function sends" is specific to this example, but I could see a developer interpreting it as "a confirmation function sends a message that the user must respond to" which isn't true inherently. It could be doing anything. Maybe we say instead something like:

> Unless the custom function automatically confirms the user, you must provide a
> means for the user to complete the confirmation process after the function fires.
> After completing any additional confirmation steps, call the SDK's Confirm User 
> method to finalize the user account creation on Atlas App Services.

Reply:
much better!